frappe/utils/remote_migrations.py

```diff
@@ -0,0 +1,269 @@
+# imports - standard imports
+import functools
+import getpass
+import json
+import re
+import sys
+
+# imports - third party imports
+import click
+from html2text import html2text
+import requests
+
+# imports - module imports
+import frappe
+import frappe.utils.backups
+from frappe.utils import get_installed_apps_info
+from frappe.utils.commands import render_table, padme
+
+
+def get_new_site_options():
```

Comment:
Move methods specific to frappecloud migration to `frappe_providers/frappecloud.py`, then we can have different files for different providers.